packages/ember-routing/lib/system/route.js

```diff
@@ -1878,7 +1878,8 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
     The string values provided for the template name, and controller
     will eventually pass through to the resolver for lookup. See
     Ember.Resolver for how these are mapped to JavaScript objects in your
-    application.
+    application. The template needs to be either the one of the current route or one
```

Comment:
Should this be "The template to render into needs to be" (so that it's clear that these rules only apply to the `into` template and not to the template we want to render)?